README.md

```diff
@@ -348,6 +348,8 @@ An array of folders. Resolved modules only from those folders will be considered
 
 This option is also useful in a monorepo setup: list here all directories that contain monorepo's packages and they will be treated as external ones no matter which resolver is used.
 
+Also, this option can be used with plug'n'play mode. For example, if you are using `yarn berry` you can set `.yarn` folder and all your installed dependencies will be considered as `external`, otherwise it will be `internal`.
```

Comment:
most users will have no idea what "plun'n'play mode" is, let alone yarn. What about:
```suggestion
If you are using `yarn` PnP as your package manager, add the `.yarn` folder and all your installed dependencies will be considered as `external`, instead of `internal`.
```

Reply:
Sounds good, fixed